src/services/upload.services.js

```diff
@@ -33,6 +33,39 @@ class UploadService {
 		return blobLink;
 	};
 
+	// FIXME
+	// if move url image from content to thumbnail that lead to in the future when 
+	// query the list image of the content will miss the once which was converted
+	// to thumbnail
+	static uploadImageUrl = async (req) => {
+		let userId = req.cookies.userId;
+		let { topic, postId } = req.query;
+		let { url } = req.body;
+		if (!userId) {
+			throw new AuthFailureError({
+				message: 'Please verify your authentication',
+			});
+		}
+		if (!url) {
+			throw new BadRequestError({
+				message: 'Please Giving The Image Url',
+			});
+		}
+		if (!AVATAR_TOPIC.includes(topic)) {
+			throw new BadRequestError({
+				message: 'Please give correct topic',
+			});
+		} else if (topic == 'thumnail' && !postId) {
```

Comment:
I think the topic is content we should give postId too
if(topic == 'thumnail' || topic == 'content' && !postId)